starter-code/main.js

```diff
@@ -6,84 +6,278 @@ const rl = readline.createInterface({ input: process.stdin, output: process.stdo
+
+					case '9':
+					// 9 .- How many employees has Facebook?
+						db.collection('companies').find( {name: 'Facebook'},{  number_of_employees: 1, _id: 0}).toArray((error, result) => {
+							if (error){
+								console.log(error)
+								rl.question(`\nType enter to continue: `, (answer) => { mainMenu() })
+							} else {
+								console.log(result)
+								rl.question(`\nType enter to continue: `, (answer) => { mainMenu() })
+							}
+						})
+					break;
+
+					case '10':
+					// 10 .- List the name of all the products of Facebook
+						db.collection('companies').find( {name: 'Facebook'},{ 'products.name':1, _id: 0}).toArray((error, result) => {
+							if (error){
+								console.log(error)
+								rl.question(`\nType enter to continue: `, (answer) => { mainMenu() })
+							} else {
+								console.log(...result)
+								rl.question(`\nType enter to continue: `, (answer) => { mainMenu() })
+							}
+						})
+					break;
+
+					case '11':
+					// 11 .- List the people that are working at Facebook right now (check relationships field)
+					db.collection('companies').find({$and: [{name: 'Facebook'}, {'relationships.is_past': false}]},{ 'relationships.person.first_name':1, _id: 0}).toArray((error, result) => {
+						if (error){
+							console.log(error)
+							rl.question(`\nType enter to continue: `, (answer) => { mainMenu() })
+						} else {
+							console.log(result[0].relationships)
+							rl.question(`\nType enter to continue: `, (answer) => { mainMenu() })
+						}
+					})
+					break;
+
+					case '12': 
+					// 12 .- List all the companies where "david-ebersman" has worked.
+					db.collection('companies').find({$and: [{},{'relationships.person.permalink': 'david-ebersman'}]},{ 'name':1, _id: 0}).toArray((error, result) => {
+						if (error){
+							console.log(error)
+							rl.question(`\nType enter to continue: `, (answer) => { mainMenu() })
+						} else {
+							console.log(result)
+							rl.question(`\nType enter to continue: `, (answer) => { mainMenu() })
+						}
+					})
+					break;
+
+					case '13':
+					// 13 .- List by name the competitors of Facebook
+					db.collection('companies').find({name: 'Facebook'},{ 'competitions.competitor.name':1, _id: 0}).toArray((error, result) => {
+						if (error){
+							console.log(error)
+							rl.question(`\nType enter to continue: `, (answer) => { mainMenu() })
+						} else {
+							resultFinal = result[0].competitions
+							for (let i = 0; i < resultFinal.length; i++) {
+								console.log(resultFinal[i].competitor.name)
+							}
+							rl.question(`\nType enter to continue: `, (answer) => { mainMenu() })
+						}
+					})
+					break;
+
+					case '14':
+					// 14 .- Names of the companies that has "social-networking" in tag-list (be aware that the value of field is a string check regex operators)
+					db.collection('companies').find({$and: [{},{tag_list: /social-networking/}]},{ 'name':1, _id: 0}).toArray((error, result) => {
```

Comment:
Condición de búsqueda vacía:
```suggestion
					db.collection('companies').find({$and: [{tag_list: /social-networking/}]},{ 'name':1, _id: 0}).toArray((error, result) => {
```